quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/CLITestExtension.java

```diff
@@ -97,10 +105,12 @@ public void afterEach(ExtensionContext context) throws Exception {
     private void reset() {
         QuarkusConfigFactory.setConfig(null);
         //remove the config file property if set, and also the profile, to not have side effects in other tests.
-        System.getProperties().remove(KeycloakPropertiesConfigSource.KEYCLOAK_CONFIG_FILE_PROP);
         System.getProperties().remove(Environment.PROFILE);
```

Comment:
Since you are at it, can you move those property settings all in `configureDevServices` to have a uniform way of setting/resetting?

Reply:
These are not related to dev services. That is why I didn't add there.

Reply:
ok